python/shotgun_model/data_handler_nav.py

```diff
@@ -146,10 +152,40 @@ class and the data payload from Shotgun should be provided via the
         num_modifications = 0
 
         new_uids = set()
+
+        # a list of sg data dicts to display items for
+        child_data = []
+
+        if item_path == self.__root_path:
+            self._log_debug("This is the root of the tree.")
+            parent_uid = None
+
+            if self.__include_root:
```

Comment:
i agree with @jfboismenu that prior to having this special node, clicking the home button when it's linked to the project context should show the treeview (say in the loader) unselected and with all children visible - but with this change it feels like the new special root path should be selected.